Directory.Packages.props

```diff
@@ -63,8 +63,8 @@
     <PackageVersion Include="Microsoft.TemplateEngine.Orchestrator.RunnableProjects" Version="$(MicrosoftTemplateEngineOrchestratorRunnableProjectsPackageVersion)" />
     <PackageVersion Include="Microsoft.TemplateEngine.TestHelper" Version="$(MicrosoftTemplateEngineTestHelperPackageVersion)" />
     <PackageVersion Include="Microsoft.TemplateSearch.Common" Version="$(MicrosoftTemplateSearchCommonPackageVersion)" />
-    <PackageVersion Include="Microsoft.Testing.Platform" Version="$(MicrosoftTestingPlatformVersion)" />
-    <PackageVersion Include="Microsoft.Testing.Extensions.TrxReport" Version="$(MicrosoftTestingPlatformVersion)" />
+    <PackageVersion Include="Microsoft.Testing.Platform.DotNetTestClient" Version="$(MicrosoftTestingPlatformDotNetTestClientVersion)" />
+    <PackageVersion Include="Microsoft.Testing.Extensions.TrxReport" Version="$(MicrosoftTestingPlatformDotNetTestClientVersion)" />
```

Comment:
Does it mean trx report also now needs to be SB?

Reply:
idk I did not add that dependency and was wondering the same thing. Looks like that package might be unused: 
![image](https://github.com/user-attachments/assets/930ef0bc-993c-4038-8730-bfb2de4879c4)

Reply:
You got me scared for a second :P